src/output/grid.rs

```diff
@@ -43,12 +44,23 @@ impl<'a> Render<'a> {
         for file in &self.files {
             let filename = self.file_style.for_file(file, self.theme);
             let contents = filename.paint();
+            let mut width;
+
+            match filename.options.embed_hyperlinks {
+                EmbedHyperlinks::On => width = filename.bare_width(),
+                EmbedHyperlinks::Off => width = *contents.width(),
+            };
+
+            match filename.options.show_icons {
+                ShowIcons::On(_) => width += 1,
+                ShowIcons::Off => width += 0,
+            };
```

Comment:
If `EmbedHyperlinks::Off`, then we are using `*contents.width()`, which if I understand correctly *already includes the icon*. But if `ShowIcons::On(_)`, then we are adding an *additional* 1 to the width. 

Shouldn't we only add 1 when `EmbedHyperlinks::On`?

Reply:
I think you're right!

Can you sanity check that the force push fixed this?